osquery/database/tests/database_tests.cpp

```diff
@@ -78,6 +134,24 @@ TEST_F(DatabaseTests, test_delete_values) {
   EXPECT_TRUE(value.empty());
 }
 
+TEST_F(DatabaseTests, test_delete_values_int) {
+  int expected = 0;
+  setDatabaseValue(kLogs, "k", 0);
```

Comment:
1. You probably mean `setDatabaseValue(kLogs, "k", expected);` here.
2. It  is better to use non-default  value here. Like 42 or whatever :)